x-pack/plugins/osquery/public/shared_components/osquery_results/types.ts

```diff
@@ -5,25 +5,20 @@
  * 2.0.
  */
 
-import type { EcsSecurityExtension } from '@kbn/securitysolution-ecs';
+import type { Ecs } from '@kbn/cases-plugin/common';
 import type { ActionEdges } from '../../../common/search_strategy';
 
 export interface OsqueryActionResultsProps {
   ruleName?: string[];
-  ecsData: EcsSecurityExtension;
+  ecsData?: Ecs | null;
   actionItems?: ActionEdges;
 }
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
 
 export interface OsqueryActionResultProps {
   ruleName?: string[];
-  ecsData: EcsSecurityExtension;
+  ecsData?: Ecs | null;
   actionId: string;
   queryId: string;
   startDate: string;
+  isNewFlyout?: boolean;
```

Comment:
just a thought, but I would rename `isNewFlyout` to `isExpandableFlyout` everywhere in this PR because **_new_** means something now (while still less descriptive than `isExpandableFlyout`), but won't mean much in a few months when we release everything